### PR TITLE
[Author profile] Ibrahim Bougaoua with new plugin.

### PR DESCRIPTION
**Pull request by ibrahimBougaoua**

### CODE REVIEW

File: content/plugins/ibrahimBougaoua-sort-order.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Sort Order
+slug: sort-order
```
Comment (danharrin) [inline content/plugins/ibrahimBougaoua-sort-order.md:3]:
```suggestion
slug: ibrahim-bougaoua-sort-order
```

File: content/plugins/ibrahimBougaoua-sort-order.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+author_slug: awcodes
```
Comment (danharrin) [inline content/plugins/ibrahimBougaoua-sort-order.md:4]:
```suggestion
author_slug: ibrahim-bougaoua
```

File: content/authors/ibrahimBougaoua.md
Comment (danharrin) [inline content/authors/ibrahimBougaoua.md:1]: Should use ibrahim-bougaoua

File: content/plugins/ibrahimBougaoua-sort-order.md
Comment (danharrin) [inline content/plugins/ibrahimBougaoua-sort-order.md:1]: Should use ibrahim-bougaoua

File: content/plugins/ibrahimBougaoua-sort-order.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+categories: [action, table-column]
+description: About Transform the sorting order of any table effortlessly by installing this package. It provides seamless functionality without requiring any manual code writing. Simply install it, and you're good to go !
```
Comment (danharrin) [inline content/plugins/ibrahimBougaoua-sort-order.md:6]:
```suggestion
description: Transform the sorting order of any table effortlessly.
```

File: content/plugins/ibrahimBougaoua-sort-order.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+discord_url: 
+docs_url: https://raw.githubusercontent.com/ibrahimBougaoua/filament-sort-order/main/README.md
+github_repository: ibrahimBougaoua/filament-sort-order
+has_dark_theme: true
+has_translations: false
+image: sort-order.png
+thumbnail: sort-order.jpg
```
Comment (danharrin) [inline content/plugins/ibrahimBougaoua-sort-order.md:13]:
```suggestion
```

File: content/plugins/ibrahimBougaoua-sort-order.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+slug: sort-order
+author_slug: awcodes
+categories: [action, table-column]
+description: About Transform the sorting order of any table effortlessly by installing this package. It provides seamless functionality without requiring any manual code writing. Simply install it, and you're good to go !
+discord_url: 
+docs_url: https://raw.githubusercontent.com/ibrahimBougaoua/filament-sort-order/main/README.md
+github_repository: ibrahimBougaoua/filament-sort-order
+has_dark_theme: true
+has_translations: false
+image: sort-order.png
```
Comment (danharrin) [inline content/plugins/ibrahimBougaoua-sort-order.md:12]:
```suggestion
image: ibrahim-bougaoua-sort-order.png
```

File: content/plugins/thumbnails/sort-order.png
Comment (danharrin) [inline content/plugins/thumbnails/sort-order.png:1]: This can be removed as it is identical to the image

File: content/plugins/images/sort-order.png
Comment (danharrin) [inline content/plugins/images/sort-order.png:1]: This should be prefixed with ibrahim-bougaoua

**Review by danharrin — CHANGES_REQUESTED** [review]

File: content/plugins/images/sort-order.png
Comment (ibrahimBougaoua) [reply content/plugins/images/sort-order.png:1]: It's Done.

**Pull request merged**